### PR TITLE
Browse Workflow table and MyHub section css fix, logs section revamp and minor KUI fixes

### PR DESCRIPTION
**Pull request by amityt**
## Proposed changes

Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

## Special notes for your reviewer:
Screenshot from 2021-01-15 12-48-32
Screenshot from 2021-01-15 12-51-28
Screenshot from 2021-01-15 13-54-36

### CODE REVIEW

**Review by Jonsy13 — APPROVED** [review]

**Review by S-ayanide — APPROVED** [review]

File: litmus-portal/frontend/src/views/ChaosWorkflows/BrowseSchedule/styles.ts [hunk 1]
```diff
@@ -43,7 +42,7 @@ const useStyles = makeStyles((theme) => ({
   },
   tableMain: {
     marginTop: theme.spacing(4.25),
-    border: `1px solid ${theme.palette.border.main}`,
+    border: '1px solid rgba(0, 0, 0, 0.05)',
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/views/ChaosWorkflows/BrowseSchedule/styles.ts:45]: can we use disabled background here?

**Review by ishangupta-ds — DISMISSED** [review]
PTAL at border colours @amityt cc: @S-ayanide

File: litmus-portal/frontend/src/views/ChaosWorkflows/BrowseSchedule/styles.ts (hunk 1 above)
Comment (amityt) [reply litmus-portal/frontend/src/views/ChaosWorkflows/BrowseSchedule/styles.ts:45]: Sure, making the changes.
Thanks @ishangupta-ds

Comment (S-ayanide) [reply litmus-portal/frontend/src/views/ChaosWorkflows/BrowseSchedule/styles.ts:45]: According to figma tables aren't using borders, the internal gird is having border main

**Review by S-ayanide — APPROVED** [review]

**Review by Jonsy13 — APPROVED** [review]

**Pull request merged**